xbmc/rendering/gles/RenderSystemGLES.cpp

```diff
@@ -183,26 +183,14 @@ bool CRenderSystemGLES::IsExtSupported(const char* extension) const
     // GLES has FBO as a core element, not an extension!
     return true;
   }
-  else if (strcmp( extension, "GL_TEXTURE_NPOT" ) == 0)
```

Comment:
Are you sure this case is covered by the else block?

Reply:
a quick grep of the code doesn't show that we explicitly check for `GL_TEXTURE_NPOT` anywhere